docs/topics/contracts.rst

```diff
@@ -1,7 +1,7 @@
 .. _topics-contracts:
 
 =================
-Spiders Contracts
+Spider Contracts
 =================
```

Comment:
Please, also edit the `===…` before and after accordingly, i.e. remove 1 `=` from each.